specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json

```diff
@@ -1259,8 +1259,8 @@
           "type": "string",
           "readOnly": true,
           "x-nullable": false,
-          "title": "Specifies the provisioning state of the file server.",
-          "description": "Possible values: creating - The fileServer is just getting created. Updating - The file server creation has been accepted and it is getting updated. Deleting - The user has requested that the cluster be deleted, but the delete operation has not yet completed. Failed - The file server creation has failed with the specified errorCode. Details about the error code are specified in the message field. Succeeded - The file server creation has succeeded.",
+          "title": "Specifies the provisioning state of the File Server.",
+          "description": "Possible values: creating - The File Server is getting created. Updating - The File Server creation has been accepted and it is getting updated. Deleting - The user has requested that the File Server be deleted, and it is in the process of being deleted. Failed - The File Server creation has failed with the specified errorCode. Details about the error code are specified in the message field. Succeeded - The File Server creation has succeeded.",
```

Comment:
Does "creating" need to be capitalized (in the "Possible values: creating" part)?  The other values are capitalized.

Reply:
thanks, that's the opposite. fixed